authz/sdk_server_interceptors.go

```diff
@@ -32,6 +32,8 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+const logLevel = 2
```

Comment:
I don't think we should have this const; just declare the verbosity per-log-line.

Reply:
I removed the logs from sdk_server_interceptors and also this const variable.